torch/csrc/jit/python/script_init.cpp

```diff
@@ -1369,11 +1369,30 @@ void initJitScriptBindings(PyObject* module) {
       [](const std::string& qualifiedName,
          const ClassDef& classDef,
          ResolutionCallback rcb,
-         bool is_module) {
+         bool is_module,
+         py::handle py_ignored_argment_names) {
```

Comment:
Optional:
You can provide a default value for py_ignored_argument_names so that this change is completely BC, just in case anyone is using this internal method.

Nit:
s/argment/argument